materialsLibrary/materials/water/babylon.waterMaterial.ts

```diff
@@ -695,6 +695,17 @@ module BABYLON {
             if (this.bumpTexture) {
                 this.bumpTexture.dispose();
             }
+
+            var index = this.getScene().customRenderTargets.indexOf(this._refractionRTT);
+            if (index != -1){
+                this.getScene().customRenderTargets.splice(index, 1);
+            }
+            index = -1;
+            index = this.getScene().customRenderTargets.indexOf(this._reflectionRTT);
```

Comment:
Once the commit is merged, I'll add the renderlist to the serialize and parse methods